Doc/c-api/structures.rst

```diff
@@ -294,3 +294,43 @@ definition with the same method name.
    read-only access.  Using :c:macro:`T_STRING` for :attr:`type` implies
    :c:macro:`READONLY`.  Only :c:macro:`T_OBJECT` and :c:macro:`T_OBJECT_EX`
    members can be deleted.  (They are set to *NULL*).
+
+
+.. c:type:: PyGetSetDef
+
+   Structure to define property-like access for a type. See also description of
+   the :c:member:`PyTypeObject.tp_getset` slot.
+
+   +-------------+------------------+-----------------------------------+
+   | Field       | C Type           | Meaning                           |
+   +=============+==================+===================================+
+   | name        | const char \*    | attribute name.                   |
```

Comment:
If the description ends with a period, it should starts with an upper case. But descriptions in other tables don't start with an upper case and don't contain periods. Please make the style of this table matching the style of other tables.

The only problem is with the `set` description, which consists of two sentences. I think you can use a semicolon for separating two parts.